### PR TITLE
Fix wrong error message when too short password

### PR DESCRIPTION
**Pull request by yguedidi**

### CODE REVIEW

File: src/Validator/Constraints/PasswordStrengthValidator.php [hunk 1]
```diff
@@ -139,7 +139,6 @@ public function validate($password, Constraint $constraint)
 
         if ($passwordStrength < $constraint->minStrength) {
             $parameters = array(
-                '{{ length }}' => $constraint->minLength,
```
Comment (stof) [inline src/Validator/Constraints/PasswordStrengthValidator.php]: This is a BC break, as people could rely on it in custom messages (even though it is not really useless)

**Comment by sstok** [comment]
Will make a new release in a few minutes as I noticed the Dutch translations is also wrong :smile:

**Comment by stof** [comment]
@sstok Please include the French translations in the release too :smile:

**Comment by sstok** [comment]
No problem :+1:

**Comment by yguedidi** [comment]
@sstok Thanks!

**Pull request merged**